functions/ecom.config.js

```diff
@@ -256,6 +256,16 @@ const app = {
             format: 'uri',
             title: 'Ícone',
             description: 'Ícone customizado para a forma de pagamento, URL da imagem'
+          },
+          bankSlipType: {
+            type: 'string',
+            enum: [
+              'bank_slip',
+              'bank_slip_yapay'
+            ],
+            default: 'bank_slip_yapay',
+            title: 'Escolha o tipo de boleto utilizado',
+            description: 'Caso utilize Yapay, deverar selecionar a opçao com o descritivo referente, caso seja qualquer outra modalidade de boleto, deverá ser a outra opção'
```

Comment:
Se for pra ser um enum as opções deverima ser mais descritivas para o usuário, como já mencionei com você em config de outro app inclusive, e então na função você usa um switch case.

Mas de qualquer forma nem acho que deveria ser um enum aqui, só um booleano (checkbox) para o caso de ser boleto Yapay, já que é o único caso fora do padrão.

Reply:
```suggestion
          },
          is_yapay: {
            type: 'boolean',
            default: false,
            title: 'Boleto Yapay'
```